crates/holochain/tests/websocket/mod.rs

```diff
@@ -982,3 +982,65 @@ async fn holochain_websockets_listen_on_ipv4_and_ipv6() {
         _ => panic!("unexpected response"),
     }
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn emit_signal_after_app_connection_closed() {
```

Comment:
Another slightly indirect way to test this would be to write a custom tracing subscriber to assert this one message was logged. Some [inspiration](https://github.com/holochain/holochain/blob/9a8390f789fee98616c49b235fec1751025073f1/crates/aitia/src/logging.rs#L196-L217) here.

Reply:
That would be a step in the right direction but it would only let me assert that the outbound task was shutting down. I would like to assert that all the tasks allocated to this connection shut down so that we know we aren't holding onto any resources we can't use.